cookbook/configuration/pdo_session_storage.rst

```diff
@@ -172,10 +229,10 @@ following (MySQL):
 .. code-block:: sql
 
     CREATE TABLE `session` (
-        `session_id` VARBINARY(128) NOT NULL PRIMARY KEY,
-        `session_data` BLOB NOT NULL,
-        `session_time` INTEGER UNSIGNED NOT NULL,
-        `session_lifetime` MEDIUMINT NOT NULL
+        `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+        `sess_data` BLOB NOT NULL,
```

Comment:
> I think something should be said about what happens if the data limit is exceeded: the session is reset without warning or error...

This depends of the configuration of your database server actually. If you run MySQL in strict SQL compliance mode, it will return an error when saving the session instead of silently accepting the invalid value and dropping it